exporters/geneva/include/opentelemetry/exporters/geneva/metrics/exporter_options.h

```diff
@@ -0,0 +1,28 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "opentelemetry/version.h"
+#include <string>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter {
+namespace geneva {
+namespace metrics {
+
+struct ExporterOptions {
+  /*
+  Format -
+    Windows:
+        Account={MetricAccount};NameSpace={MetricNamespace}
+    Linux:
+        Endpoint=unix://{UDS
```

Comment:
nit: probably don't break/format the line here for better help?

Reply:
Yeah good point. Have disable clang format for this part of code.